### ISSUE TITLE
Determinism issues possibly due to frontend issues

### ISSUE THREAD

**Issue by nattthebear** [TASVideos/BizHawk#2522]
This is a lot of tickets, and by gathering them all together here I'm not stating with certainty that _all_ of them are the same issue, or even that _all_ of them are frontend issues.  The truth is always more nuanced than that.  But this is a lot of issues, all of them recently created.  A small number of users reported most of them, suggesting that those users may inadvertently "trigger" things with their style of usage.  Some of the referenced cores haven't seen any major work in a long time, so it seems unlikely that those cores are broken.

I want to investigate and fix core determinism issues, but the water is very muddy right now.

https://github.com/TASVideos/BizHawk/issues/2498
https://github.com/TASVideos/BizHawk/issues/2496
https://github.com/TASVideos/BizHawk/issues/2484
https://github.com/TASVideos/BizHawk/issues/2473
https://github.com/TASVideos/BizHawk/issues/2470
https://github.com/TASVideos/BizHawk/issues/2469
https://github.com/TASVideos/BizHawk/issues/2452
https://github.com/TASVideos/BizHawk/issues/2438
https://github.com/TASVideos/BizHawk/issues/2437
https://github.com/TASVideos/BizHawk/issues/2406
https://github.com/TASVideos/BizHawk/issues/2388
https://github.com/TASVideos/BizHawk/issues/2384

**Comment by YoshiRulz** [comment]
This makes a lot of sense actually. I think #2484 should be priority number 1 for 2.5.3.

**Comment by ThunderAxe31** [comment]
Please include this in the list: https://github.com/TASVideos/BizHawk/issues/2214

**Comment by nattthebear** [comment]
> 
> 
> Please include this in the list: #2214

Agreed and done.  Since that ticket has a whole lot of "not sure, can't reproduce" in it, it _might_ be frontend issues.
